installation/routines/customize_options.sh

```diff
@@ -283,47 +283,43 @@ Disable Pi's on-chip audio (headphone / jack output)? [y/N]"
 _option_webapp_devel_build() {
   # Let's detect if we are on the official release branch
   if [[ "$GIT_BRANCH" != "${GIT_BRANCH_RELEASE}" || "$GIT_USER" != "$GIT_UPSTREAM_USER" || "$CI_RUNNING" == "true" ]]; then
-    ENABLE_INSTALL_NODE=true
     # Unless ENABLE_WEBAPP_PROD_DOWNLOAD is forced to true by user override, do not download a potentially stale build
     if [[ "$ENABLE_WEBAPP_PROD_DOWNLOAD" == "release-only" ]]; then
       ENABLE_WEBAPP_PROD_DOWNLOAD=false
     fi
-    if [[ "$ENABLE_WEBAPP_PROD_DOWNLOAD" == false ]]; then
+    if [[ "$ENABLE_WEBAPP_PROD_DOWNLOAD" != true && "$ENABLE_WEBAPP_PROD_DOWNLOAD" != "release-only" ]]; then
       clear_c
-      print_c "--------------------- WEBAPP NODE ---------------------
+      print_c "--------------------- WEBAPP BUILD ----------------------
 
 You are installing from an unofficial branch.
 Therefore a prebuilt web app is not available and
-you will have to build it locally.
+it needs to be build locally.
 This requires Node to be installed.
 
-You can choose to decline the Node installation and
-the lastest prebuilt version from the main repository
-will be installed. This can lead to incompatibilities.
+You can choose to decline and the lastest prebuilt
```

Comment:
`and` should be removed to make the sentence more readable

Reply:
Rephrased it